static/js/util/util.js

```diff
@@ -546,3 +546,11 @@ export function sortedCourseRuns(program: Program): Array<CourseRun> {
     R.map(R.sortBy(R.prop('position')), R.pluck('runs', sortedCourses))
   );
 }
+
+// lets you edit both keys and values
+// just pass a function that expects and returns [key, value]
+export const editObj = R.curry((fn, obj) => R.compose(
```

Comment:
nit: I feel like `objMap` might be a better name since the intent is to create an altered copy of the object, not edit it in place